transports/sqs/delayed-delivery.md

```diff
@@ -0,0 +1,145 @@
+
+snippet: DelayedDelivery
+
+Unrestricted delayed delivery needs to be enabled both on the sender and receiver to be able to process delayed deliveries longer than 900 seconds. Delayed delivery below 900 seconds is always supported. The following table illustrates that.
+
+| Scenario                    | Sender   | Receiver | Supported     |
+|-----------------------------|----------|----------|---------------|
+| Delayed Delivery <= 900 sec | disabled | disabled | Yes           |
+|                             | disabled | enabled  | Yes           |
+|                             | enabled  | disabled | Yes           |
+|                             | enabled  | enabled  | Yes           |
+| Delayed Delivery > 900 sec  | disabled | disabled | No            |
+|                             | disabled | enabled  | No            |
+|                             | enabled  | disabled | No            |
+|                             | enabled  | enabled  | Yes           |
+
+Enabling the unrestricted delayed delivery will require a FIFO queue to be created for each endpoint that receives delayed deliveries. The FIFO queue follows a fixed naming convention by appending `-delay.fifo` to the queue name of the endpoint. The creation of the FIFO queue requires the [installers](/nservicebus/operations/installers.md) to be enabled or the queue being created upfront via [scripting](/transports/sqs/operations-scripting.md).
+
+NOTE Unrestricted delayed delivery works seamlessly with large message bodies.
+
+## How it works
+
+Each endpoint with unrestricted delayed delivery owns a FIFO queue that is used to offload delayed messages from the main input queue until they are due. The FIFO queue uses a fixed delay interval of 900 seconds on the queue level. The FIFO queue shields the timeout dispatching mechanism in a time window of five minutes from infinitely growing timeouts due to network outages between the timeout requeueing and removing the previous timeout from the queue.
+
+When a sender sends a delayed message to a destination, it determines whether the delay is less or equal to 900 seconds. If that is the case, the message is directly delayed to the destination queue by setting the `DelaySecond` behavior on the message. When the timeout is greater than 900 seconds, the message is sent to the FIFO queue of the destination endpoint with a message attributed called `NServiceBus.AmazonSQS.DelaySeconds`. The delayed message consumer on the destination's FIFO queue receives all timeouts that are due after 900 seconds. When a timeout is due, and the remaining delay is less or equal to 900 seconds the message is directly delayed to the destination queue. If the remaining delay is greater than 900 seconds, the message is sent back to the FIFO queue containing a `DelaySeconds` attribute with the remaining timeout interval. The following sequence diagram illustrates the process:
```

Comment:
> the message is sent to the FIFO queue of the destination endpoint with a message attributed called `NServiceBus.AmazonSQS.DelaySeconds`.

typo: attribute**d**

And it should be probably "attribute named" instead of "called"

Reply:
> and the remaining delay is less or equal to 900 seconds

is missing a `than`:

> and the remaining delay is less **than** or equal to 900 seconds